scripts/release.js

```diff
@@ -154,11 +181,13 @@ logger.warn(`
 shouldContinuePrompt();
 
 // Creates a new React test app, install component and sdk and build the test app to ensure no peer dependency warnings, errors or build issues
-logger.log('Create a new npm package and install amazon-chime-sdk-component-library-react and amazon-chime-sdk-js sdk as dependencies, check if there is peer dependency warning for amazon-chime-sdk-component-library-react');
+logger.log(
+  'Create a new npm package and install amazon-chime-sdk-component-library-react and amazon-chime-sdk-js sdk as dependencies, check if there is peer dependency warning for amazon-chime-sdk-component-library-react'
+);
 process.chdir(path.join(__dirname, '..'));
 spawnOrFail('yalc', ['publish']);
```

Comment:
Since you use npm pack elsewhere, can you remove yalc here and just use npm pack and npm install?

Reply:
So for the `spawnOrFail('yalc', ['publish']);`, will do
```
spawnOrFail('npm', ['run build']);
spawnOrFail('npm', ['pack']);
```
And for `spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);`, will do
```
spawnOrFail('npm', [`install amazon-chime-sdk-component-library-react-${versionString}.tgz`,]);
```
Is it correct?